### PR TITLE
I281 clean up documentation and readme

### LINKED ISSUE #281: Clean up documentation and readme
## Basic Information
**[Give a brief summary of what the issue is about]**
- add info about accessing docs in readme
- Some endpoints and fields have been updated recently and documentation hasnt been updated :(

### LINKED ISSUE #283: Avatar images not displaying on production
## Basic Information
Only images in subteam dropdown (on teams page)
- Is it using the images in public folder?

### PR DESCRIPTION
**Pull request by ChantelleKerr**
## Change Summary

**[Briefly summarise the changes that you made. Just high-level stuff]**
- done but i forgot why i needed to update readme?

### Change Form

**Fill this up (NA if not available). If a certain criteria is not met, can you please give a reason.**

- [ ] The pull request title has an issue number
- [ ] The change works by "Smoke testing" or quick testing
- [ ] The change has tests
- [ ] The change has documentation

## Other Information

**[Is there anything in particular in the review that I should be aware of?]**

Resolve #283 
Resolve #281 

# Related Issue

- Resolve #281

### CODE REVIEW

File: documentation/docs/admin/accounts.md [hunk 1]
```diff
@@ -26,7 +26,9 @@
 - `Team id`: Used to select the team this user is part of (can be blank)
 - `Team admin`: This should be checked if the user is the admin of the team above
 - `Reset Token`: Ignore
-- `Reset Time`: Sets challenge reset date and time (leave this as blank)
+- `Reset Time`: Ignore - Time the token should reset
+- `challenge_start_date`: Ignore - Sets challenge reset date and time (leave this as blank)
```
Comment (RadioArsenic) [inline documentation/docs/admin/accounts.md:30]: I think the descriptions for lines 28-30 are all mixed around

File: documentation/docs/backend/email.md [hunk 2]
```diff
@@ -1,5 +1,15 @@
...
-MailHog is being used to test the sending of emails. It is an email-testing tool with a fake SMTP server underneath. To view the emails which have been sent, open `http://localhost:8025/` in your browser, which will open the MailHog user interface.
+### Development
+
+MailHog is being used to test the sending of emails for free in a development environment. It is an email-testing tool with a fake SMTP server underneath. To view the emails which have been sent, open `http://localhost:8025/` in your browser, which will open the MailHog user interface.
 
 If you haven't got it installed already, Mailhog should automatically install when you run `docker compose up`.
+
+### Production
+
+Sendgrind is used to send and recieve emails in the production version of the application. Community Spirit Foundation email account has been added as the sender. The Sendgrid account is made under Coders For Causes credentials and will allow for 100 free emails per day, anything over will cost $20 per month.
```
Comment (RadioArsenic) [inline documentation/docs/backend/email.md:11]: - Isn't it "Sendgrid" not "Sendgrind" xD 
- Add "The" before  "Community Spirit Foundation email account"

File: documentation/docs/backend/email.md [hunk 3]
```diff
@@ -1,5 +1,15 @@
...
+
+Password reset emails will be sent from the Community Spirit Foundation to email address to the user email that requests the change.
```
Comment (RadioArsenic) [inline documentation/docs/backend/email.md:13]: remove the "to" between "Community Spirit Foundation" and "email address"

File: documentation/docs/backend/email.md [hunk 4]
```diff
@@ -1,5 +1,15 @@
...
+
+Everytime a new user has created an account the Community Spirit Foundation will recieve an email to let admin know.
```
Comment (RadioArsenic) [inline documentation/docs/backend/email.md:15]: add "email address" after "Community Spirit Foundation"

Also it's "receive" not "recieve"

File: documentation/docs/backend/endpoints.md [hunk 5]
```diff
@@ -60,11 +66,13 @@
...
 - `kilometers`
 - `date`
+- `team`
 
 #### Views
 
-- `create_mileage/` - create new mileage
-- `get_mileage/<int:user_id>` - Retrieve a list of mileage by user_id
+- `get_leaderboard/` - Returns leaderboard data
+- `get_mileage` - Retrieve a list of mileage
```
Comment (RadioArsenic) [inline documentation/docs/backend/endpoints.md:74]: Say "Returns" not "Retrieve" in line 74

**Review by RadioArsenic — CHANGES_REQUESTED** [review]

File: documentation/docs/admin/accounts.md (hunk 1 above)
Comment (ChantelleKerr) [reply documentation/docs/admin/accounts.md:30]: reset time is about the token - I dont think its currently used anywhere tbh

Comment (RadioArsenic) [reply documentation/docs/admin/accounts.md:30]: Can you reword that to make it more clear? Such as "Sets a time for the token above to reset".
Also is challenge start date used to "set the challenge reset date and time" or "used to calculate the challenge reset date and time", the latter makes more sense.

File: documentation/docs/backend/email.md (hunk 2 above)
Comment (RadioArsenic) [reply documentation/docs/backend/email.md]: Just noticed another "recieve" which should be "receive"

File: documentation/docs/backend/email.md [hunk 6]
```diff
@@ -8,8 +8,8 @@ If you haven't got it installed already, Mailhog should automatically install wh
 
 ### Production
 
-Sendgrind is used to send and recieve emails in the production version of the application. Community Spirit Foundation email account has been added as the sender. The Sendgrid account is made under Coders For Causes credentials and will allow for 100 free emails per day, anything over will cost $20 per month.
+Sendgrid is used to send and recieve emails in the production version of the application. The Community Spirit Foundation email account has been added as the sender. The Sendgrid account is made under Coders For Causes credentials and will allow for 100 free emails per day, anything over will cost $20 per month.
```
Comment (RadioArsenic) [inline documentation/docs/backend/email.md:11]: just noticed another "recieve" which should be "receive"

File: documentation/docs/backend/endpoints.md (hunk 5 above)
Comment (RadioArsenic) [reply documentation/docs/backend/endpoints.md]: "Returns" not "Return", say "Returns a list of mileage data" so it matches the wording of get leaderboard

**Review by RadioArsenic — CHANGES_REQUESTED** [review]
Plus the other two unresolved from the last review :)

**Pull request merged**